src/Sarif/Visitors/InsertOptionalDataVisitor.cs

```diff
@@ -117,8 +117,6 @@ public override PhysicalLocation VisitPhysicalLocation(PhysicalLocation node)
                     resolvedUri = artifactLocation.Uri;
                 }
 
-                if (!resolvedUri.IsAbsoluteUri) { goto Exit; }
```

Comment:
>if (!resolvedUri.IsAbsoluteUri) { goto Exit; } [](start = 16, length = 46)

Since our Uri can be relative, I'm removing this so we can generate contextRegion